packages/fast-form-generator-react/src/form/form-item.select.style.ts

```diff
@@ -0,0 +1,70 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+
+function applypaddingStyle(): any {
+    return {
+        padding: localizeSpacing(Direction.ltr)(`${toPx(8)} ${toPx(24)} ${toPx(8)} ${toPx(8)}`)
+    };
+}
+
+const styles: any = {
+    form_item_select_wrapper: {
+        display: "flex",
+        alignItems: "center",
+        flexDirection: "row",
+        paddingTop: toPx(8)
+    },
+    form_item_select_label: {
+        flexGrow: "1",
+        lineHeight: toPx(16),
+        fontSize: toPx(14),
+        marginRight: toPx(16),
+        ...ellipsis()
+    },
+    form_item_select_span: {
+        position: "relative",
+        display: "flex",
+        flexGrow: "1",
+        "&:before, &:after": {
+            content: "''",
+            position: "absolute",
+            top: toPx(12),
+            zIndex: 1,
+            borderRadius: toPx(2),
+            width: toPx(1),
+            height: toPx(10),
+            background: "black"
+        },
+        "&:before": {
+            right: toPx(15),
+            transform: "rotate(45deg)"
+        },
+        "&:after": {
+            right: toPx(22),
+            transform: "rotate(-45deg)"
+        }
+    },
+    form_item_select_input: {
```

Comment:
these names should all match eg formItemSelect_input and remove _wrapper on the object key above

Reply:
Well, not we know how this didn't error